go.mod

```diff
@@ -2,13 +2,15 @@ module sqlflow.org/sqlflow
 
 require (
 	github.com/apache/thrift v0.13.0 // indirect
+	github.com/argoproj/argo v2.4.3+incompatible
```

Comment:
Just a mention, the `argo` package  is ** incompatible** since there is no `go.mod` file in it, we should fill the dependency such as `k8s.io/go-client` in manual.

Reply:
Good catch. I am doing that right now. Also, I've filed an issue in the Argo repo to see if they are planning to move to `go.mod`.

Reply:
The Argo team is planning on migrating to `go.mod`, but it won't be done in the short term. So I would suggest using `kubectl` to fetch logs directly.